src/Components/App.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Form from './Form.jsx';
+import UserInfo from './UserInfo.jsx';
+const xxx = '?client_id=fccd37f38519b0d71cd7&client_secret=61572c304be174f925b52e267794cf5c9f768e00';
```

Comment:
trochę słaba nazwa dla stałej?

Reply:
tego miało nie być, gdyż jest to autoryzacja do gita żeby mi nie blokowało dostępu do API, w drugim pull requescie usunąłem tą stała